src/DisplayList.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DisplayList from './DisplayList';
+
+test('Has an ul',  () => {
```

Comment:
The test says it "has an ul" but I don't see anything about a 'ul' here.

Reply:
The `screen` object refers to a `ul` as a `list` so I used the `getByRole` function to grab the 'ul'. I can change the variable name on Line 9 from `list` to `ul` to make it easier to understand